src/Controls/tests/TestCases.HostApp/Issues/Issue25462.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace Maui.Controls.Sample.Issues
+{
+	[Issue(IssueTracker.Github, 25462, "StackLayout inside Scrollview with horizontal orientation not expanding", PlatformAffected.Android)]
+	public class Issue25462 : TestContentPage
+	{
+		protected override void Init()
+		{
+			Content = new VerticalStackLayout()
```

Comment:
Maybe on this page make a grid of 8 cells or something and then have the columns be orientations Horizontal/Vertical and the rows be Start/Fill/Center/End.

This way a singel screenshot can cover all the positions.